src/account_manager.cc

```diff
@@ -39,6 +39,7 @@ namespace Astroid {
 
       a->save_drafts_to = kv.second.get<string> ("save_drafts_to");
 
+      a->signature_separate = kv.second.get<bool> ("signature_separate");
```

Comment:
I had it like that first (as a string config option) which specified the separator, if it is a boolean 'signature_separate' is better, no?